### DISCUSSION TITLE
Next Avalible Slot

### DISCUSSION THREAD

**Discussion by NecromancyStudios** [SkriptLang/Skript#5266] [Suggestions] [answered]
How do i make it so that if someone breaks a block with a fortune one tool it gives them 1 of the blocks in the next inventory slot?

**Comment by NecromancyStudios** [comment]
Or make an extra drop

**Comment by abandonedaccount6235** [comment, marked as answer]
Discussions aren't intended for getting help writing scripts. Please check out the "relevant links" in the repo readme for places you can get help.
